src/t_zset.c

```diff
@@ -2663,8 +2663,15 @@ void zunionInterDiffGenericCommand(client *c, robj *dstkey, int numkeysIndex, in
         return;
     }
 
+    /* Try to allocate the src table, and abort on insufficient memory. */
+    src = ztrycalloc(sizeof(zsetopsrc) * setnum);
+    if (src == NULL) {
+        addReplyErrorFormat(c, "Insufficient memory, failed allocating transient memory for '%s' command",
+                            c->cmd->fullname);
```

Comment:
i see we do that in XAUTOCLAIM too, and there we mention the COUNT is too high.
maybe we can mention here that there are too many args?
arguably, the user already knows which command he run (i.e. this is a reply to a specific command, not a log message)

Reply:
yes, this is better, i updated it